lib/src/test/kotlin/WeatherSensorService/CustomStringConvertibleTests.kt

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
Rename according class and file to whatever the file under test was named to e.g., "DoubleExtensionsTests"